src/pid/sd-jwt/index.ts

```diff
@@ -0,0 +1,108 @@
+import base64decode from "../lib/base64";
```

Comment:
Where is `lib` folder?

Reply:
To avoid redefine some methods could base64encode/decode be imported from `io-react-native-jwt`?

Reply:
done in #7